networktest.go

```diff
@@ -984,3 +984,61 @@ func (n *networkHarness) SendCoins(ctx context.Context, amt btcutil.Amount,
 		}
 	}
 }
+
+// CopyLogs copy/dumps Alice and Bob lnd daemon logs with specified period
+// of time in temporary directory.
+// NOTE: Panics error logs will not be logged.
+func (n *networkHarness) CopyLogs(period time.Duration) (func(), <-chan error) {
```

Comment:
I am using it during debugs and I thought if might be useful to others, but essentially this is just copying of the `lnd` logs in the temp directory.